mne_bids_pipeline/steps/sensor/_02_decoding_full_epochs.py

```diff
@@ -112,6 +111,9 @@ def run_epochs_decoding(
         [epochs[epochs_conds[0]], epochs[epochs_conds[1]]], verbose="error"
     )
 
+    # Crop to the desired analysis interval.
+    epochs.crop(cfg.decoding_epochs_tmin, cfg.decoding_epochs_tmax)
```

Comment:
I don't see any problem with moving this but... how did this fix the failures ?!?

Reply:
You caught me red-handed there. I believe I'm working around a bug in MNE there. It wouldn't allow me to concatenate epochs where the baseline period was cropped away. So now I first concat, then crop.

Reply:
I think that's an okay workaround. A comment along those lines would help

Reply:
I opened an issue: https://github.com/mne-tools/mne-python/issues/12153